dev-lang/rust/rust-1.44.1.ebuild

```diff
@@ -1,9 +1,12 @@
+# Difference to upstream ebuild file:
+# - rust-crossdev.patch applied
+#
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{6,7} )
+PYTHON_COMPAT=( python3_{6,7,8} )
```

Comment:
It will not work, as our codebase still does not support `python3_8`.

Reply:
Ok, thanks, I thought this is just a range of possibilities and didn't know it would try to use it.